CRM/Core/BAO/Mapping.php

```diff
@@ -381,7 +381,7 @@ public static function buildMappingForm(&$form, $mappingType, $mappingId, $colum
 
     if (($mappingType == 'Search Builder') || ($exportMode == CRM_Export_Form_Select::CONTRIBUTE_EXPORT)) {
       if (CRM_Core_Permission::access('CiviContribute')) {
-        $fields['Contribution'] = CRM_Contribute_BAO_Contribution::exportableFields();
+        $fields['Contribution'] = CRM_Contribute_BAO_Contribution::getExportableFieldsWithPseudoConstants();
```

Comment:
This means we see both Contribution Status & Contribution Status ID in search builder. It is consistent with pre-existing behaviour